sdl_viewer/shaders/outlinedBox.fs

```diff
@@ -0,0 +1,9 @@
+#version 330 core
```

Comment:
inconsistent to other shaders (which use #version 100). I do not care which one you change, but please creating consistency.

Reply:
Also, this seems to be the same logic as points.fs - does it make sense to reuse it? Or is it more common to have files per program?

Reply:
let's use rust naming convention for files: 'outlined_box.fs'.

Reply:
fragment shader differ. color is uniform in box drawer and varying in points drawer